### ISSUE TITLE
[Config Support]: camera does not support record or stream ??

### ISSUE THREAD

**Issue by atv2016** [blakeblackshear/frigate#4967]
### Describe the problem you are having

I used (i guess with regular old, camera entities setup in my configuration.yaml) be able to stream and record these. I've since done a way with these in favour of frigate camera entities, but it seems i can't do any of these actions those.

I guess they don't have the source set for some reason (I understand this is the problem from a bit of browsing; the message i get is: This camera does not support stream or record message).

Any idea's why this is the case? I'd rather not have more camera entities again as this fills up my homepage with redundant camera entities (not to mention load up my cpu/nvr/frigate with additional RTSP feeds).

Any way of setting the source?

### Version

na

### Frigate config file

```yaml
na
```

### Relevant log output

```shell
na
```

### Frigate stats

```json
na
```

### Operating system

HassOS

### Install method

HassOS Addon

### Coral version

USB

### Any other information that may be helpful

no

**Comment by NickM-27** [comment]
Odds are this is just a matter of the integration exposing those features. I'd recommend making an issue on the integration as this is unrelated to frigate.

**Comment by atv2016** [comment]
But this is the frigate integration exposing those camera entities right ? Or am i mistaken.

**Comment by NickM-27** [comment]
Yes and the frigate integration has its own repo with its own issues https://github.com/blakeblackshear/frigate-hass-integration
